pkg/identity/identityd.go

```diff
@@ -1,13 +1,11 @@
 package identity
 
 import (
-	"encoding/json"
-	"fmt"
-	"net/http"
+	"net/url"
 
 	"github.com/rs/zerolog/log"
 
-	"github.com/threefoldtech/zos4/pkg/types"
+	"github.com/threefoldtech/tfgrid4-sdk-go/node-registrar/client"
```

Comment:
zos shouldn't use types from the client instead client should use types of zos if necessary